tests/console/validate_modify_existing_partition.pm

```diff
@@ -23,12 +23,21 @@ sub run {
     select_console "root-console";
 
     foreach my $part (keys %$test_data) {
-        record_info("Check $test_data->{fs_type}", "Verify that the partition filesystem is $test_data->{$part}->{fs_type}");
-        my $fstype = script_output("df -PT $test_data->{$part}->{mount_point} | grep -v \"Filesystem\" | awk '{print \$2}'");
+        record_info("Check $test_data->{$part}->{fs_type}", "Verify that the partition filesystem is $test_data->{$part}->{fs_type}");
+
+        my $fstype;
+        # We cannot use df for swap: it is not actually mounted but activated
+        if ("$test_data->{$part}->{fs_type}" eq "swap") {
```

Comment:
I think it is already a string, you don't need the quotes surrounding `$test_data->...`

Reply:
done